src/chrome/content/rules/thecustomizewindows.com.xml

```diff
@@ -1,9 +1,38 @@
 <ruleset name="The Customize Windows">
-<target host="thecustomizewindows.com" />
-<target host="www.thecustomizewindows.com" />
-<!-- Rackpace Cloud Files via Akamai, Jquery cookie needed to flush the client devices -->
-<securecookie host="^d9030fe1a236b276eb5c-5f19a665441657489976dbe6d75b0b9b\.ssl\.cf2\.rackcdn\.com$" name=".+"/> 
-<!-- needed for services which do not support https yet, like feedburner -->
-<exclusion pattern="^http://cloud\.thecustomizewindows\.com/"/>
-<rule from="^http://(?:www\.)?thecustomizewindows\.com/" to="https://thecustomizewindows.com/"/>
-</ruleset>
+
+		<target host="thecustomizewindows.com" />
+		<target host="www.thecustomizewindows.com" />
+		<target host="thecustomizewindows.net" />
+		<target host="www.thecustomizewindows.net" />
+
+	<!-- Akamai CDN -->
+
+		<securecookie host="^a248\.e\.akamai\.net$" name=".+"/> 
+		
+	<!-- allow trusted services -->
+	
+		<securecookie host="^cdn\.goroost\.com$" name=".+"/>
+		<securecookie host="^www\.googletagservices\.com$" 
```

Comment:
It looks like your editor has wrapped some lines. Could you change the securecookie and exclusion tags to be on a single line?

For the rule tags, it's fine to be on two lines, but the `from` tag should be aligned with the `to` tag.